service/templates/no_title_number_results.html

```diff
@@ -13,24 +13,25 @@
                     <li>Results</li>
                 </ol>
             </div>
-
+            <p/>
```

Comment:
Lack of spacing here could probably be better fixed by adding padding on the `outer-block`?